File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/35_compito_java_2019_11_20/Squadra.java

public class Squadra {

    protected int codice;
    protected String sport, nome;
    protected int vinte, perse;

    public Squadra(int codice, String sport, String nome, int vinte, int perse) {
        this.codice = codice;
        this.sport = sport;
        this.nome = nome;
        this.vinte = vinte;
        this.perse = perse;
    }

    public int getVinte() {
        return vinte;
    }
}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/09_compito_java_20160615/Squadra.py
class Squadra:
    def __init__(self, nome, codice, sport, vinte, perse):
        self._nome = nome
        self._codice = codice
        self._sport = sport
        self._vinte = vinte
        self._perse = perse
        self._giocatori = []

    def add_giocatore(self, g):
        self._giocatori.append(g)

    def stampa_squadra(self):
        print("La squadra " + self._nome + " ha " +
              str(len(self._giocatori)) + " giocatori.")

    def __str__(self):
        return self._nome + "\t" + str(self._codice) + "\t" + str(self._vinte) + "\t" + str(self._perse)
